### DISCUSSION TITLE
Is it possible to gather LHR audit/category results from Lighthouse user flow tests?

### DISCUSSION THREAD

**Discussion by gparascandolo** [GoogleChrome/lighthouse#14196] [Q&A] [answered]
Hi,

I am currently using the default node Lighthouse functionality to gather performance results of audits & categories on URLs like so
```
const {lhr} = await lighthouse(urlToAudit, flags);
console.log(`Lighthouse category scores: ${Object.values(lhr.categories).map(c => `${c.title} ${c.score}`).join(', ')}`);
```

Is it possible to also gather these metrics programmatically like above with the new Lighthouse user-flow tests introduced in v9.5.0 with the Fraggle Rock API?

At the moment, I only see the ability to generate the User Flow HTML Report and Artifacts JSON.

**Comment by adamraine** [comment, marked as answer]
It should be possible to get the results as a JSON object via `flow.createFlowResult`.

See this example:
https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md#complete-user-flow-code

The JSON is structured to include results from multiple Lighthouse runs:
https://github.com/GoogleChrome/lighthouse/blob/03af4038e0f0168c8d93b8706e8d287592704008/types/lhr/flow.d.ts#L22-L30

**Reply by gparascandolo** [reply]
Thank you for the quick response!
